pernosco-submit

```diff
@@ -39,6 +39,12 @@ upload_subparser.add_argument("--dry-run", metavar="PATH", help="Instead of uplo
 upload_subparser.add_argument("trace_dir")
 upload_subparser.add_argument("source_dirs", nargs="*")
 
+upload_package_subparser = arg_subparsers.add_parser("upload-package", help="Upload a package created using `pernosco-submit upload --dry-run' to Pernosco")
+upload_package_subparser.add_argument("--title", help="Display the given name in the Pernosco UI and tab title")
+upload_package_subparser.add_argument("--url", help="Make the name a link to the given URL")
+upload_package_subparser.add_argument("--consent-to-current-privacy-policy", action='store_true', help="Unconditionally consent to the current privacy policy")
+upload_package_subparser.add_argument("package")
```

Comment:
Add some help text for `package`? E.g. "the name of the package file produced by `pernosco-submit upload --dry-run`"

Reply:
Will do.